tests/test_deep/test_agents/test_a2c.py

```diff
@@ -19,3 +19,25 @@ def test_a2c_cnn():
     trainer = OnPolicyTrainer(algo, env, log_mode=["csv"], logdir="./logs", epochs=1)
     trainer.train()
     shutil.rmtree("./logs")
+
+
+class custom_net(MlpActorCritic):
+    def __init__(self, state_dim, action_dim, **kwargs):
+        super(custom_net, self).__init__(state_dim, action_dim, kwargs.get("hidden"))
+        self.state_dim = state_dim
+        self.action_dim = action_dim
+
+
+def test_a2c_custom():
+    env = VectorEnv("CartPole-v0", 1)
+    algo = A2C(
+        custom_net(
+            state_dim=env.observation_space.shape[0],
+            action_dim=env.action_space.n,
+            hidden=(64, 64),
+        ),
+        env,
+    )
+    trainer = OnPolicyTrainer(algo, env, log_mode=["csv"], logdir="./logs", epochs=1)
+    trainer.train()
+    shutil.rmtree("./logs")
```

Comment:
Would be better if you could shift all the custom tests to a separate file called `test_custom.py` where you can test out only for VPG (custom policy), PPO1 (custom actor-critic) and TD3 (custom actor-multicritic) maybe (as they all vary only a little bit, it would be redundant to see if all of them are running)

Reply:
Cool, I'll do this